.changes/unreleased/tinymce-TINY-11399-2024-11-06.yaml

```
@@ -0,0 +1,6 @@
+project: tinymce
+kind: Added
+body: 'The `quickimage` button now uses a`QuickbarInsertImage` command, allowing it to be registered by other plugins and replace the default action'
```

Not sure we need to state why we added it, it's mostly an internal concern?

```suggestion
body: 'The `quickimage` button now uses a `QuickbarInsertImage` command.'
```

---

done